shared-libs/infodoc/src/infodoc.js

```diff
@@ -155,8 +155,8 @@ const saveTransitions = change => {
   return saveProperty(change.id, change.info, 'transitions', {});
 };
 
-const saveCompletedTasks = (id, infodoc) => {
-  return saveProperty(id, infodoc, 'completed_tasks', {});
+const saveCompletedTasks = (id, infodoc, completedTasks = []) => {
```

Comment:
like this?

Reply:
Oh... now that I check the implementation, I see that the property value was taken from the infodoc that was already passed, my bad!